import-dashboards.py

```diff
@@ -92,6 +108,33 @@ def delete_api_key(upgrade):
     con.close()
 
 
+def fix_cloudwatch_datasource():
+    """
+    Replaces incorrect CloudWatch datasource stored as JSON string with correct JSON object.
+    """
+
+    con = sqlite3.connect(GRAFANA_DB_DIR + '/grafana.db')
+    cur = con.cursor()
+
+    found = False
+    cur.execute("SELECT id, json_data FROM data_source WHERE name = 'CloudWatch'")
+    for row in cur.fetchall():
+        found = True
+
+        old = None
+        try:
+            old = json.loads(row[1])
+        except:
+            pass
+
+        if not isinstance(old, dict):
+            new = {'authType': 'keys'}
+            cur.execute("UPDATE data_source SET json_data = ? WHERE id = ?", (json.dumps(new), row[0]))
```

Comment:
is documented credentials file way still working after this modification?
https://www.percona.com/doc/percona-monitoring-and-management/amazon-rds.html#monitoring-amazon-rds-os-metrics

Reply:
It should, but it is definitely should be tested by QA.